### PR TITLE
Minor README update and rename the text example script

### PR DESCRIPTION
**Pull request by zechengz**
- Minor README update
- Rename the text example script to `transformers_text.py` as we only use `transformers` (huggingface) in that script

### CODE REVIEW

File: docs/source/get_started/handle_text.rst [hunk 1]
```diff
@@ -128,5 +128,5 @@ Then, :obj:`stype_encoder_dict` can be directly fed into
 :class:`~torch_frame.nn.encoder.StypeWiseFeatureEncoder` to handle text columns.
 
 Please refer to the
-`pytorch-frame/examples/ft_transformer_text.py <https://github.com/pyg-team/pytorch-frame/blob/master/examples/ft_transformer_text.py>`_
+`pytorch-frame/examples/transformers_text.py <https://github.com/pyg-team/pytorch-frame/blob/master/examples/transformers_text.py>`_
```
Comment (weihua916) [inline docs/source/get_started/handle_text.rst:131]: Why did you rename it? the transformer here refers to ft_transfomer?

Comment (zechengz) [reply docs/source/get_started/handle_text.rst:131]: I want to align the name with the [llm_embedding.py](https://github.com/pyg-team/pytorch-frame/blob/master/examples/llm_embedding.py)

Comment (weihua916) [reply docs/source/get_started/handle_text.rst:131]: Got it. sounds good.

Comment (weihua916) [reply docs/source/get_started/handle_text.rst:131]: Got it. sounds good.

**Review by weihua916 — APPROVED** [review]
